src/components/TextField.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TextField extends React.Component {
+  render() {
+    const {
+      value,
+      onChange,
+      onFocus,
+      onBlur,
+      label,
+      placeholder,
+      type,
+      multiline,
+      classes = defaultClasses
+    } = this.props;
+
+    const { root, input, labelStyle } = classes;
+    const _renderTextInput = () => {
+      if (!multiline) {
+        return (
+          <div className={root}>
+            <label className={labelStyle}>{label}</label>
+            <input
+              className={input}
+              label={label}
+              onChange={onChange}
+              onBlur={onBlur}
+              onFocus={onFocus}
+              type={type}
+              value={value}
+              placeholder={placeholder}
+            />
+          </div>
+        );
+      } else {
+        return (
+          <div className={root}>
+            <label className={labelStyle}>{label}</label>
+            <textarea
+              className={input}
+              label={label}
+              onChange={onChange}
+              onBlur={onBlur}
+              onFocus={onFocus}
+              type={type}
+              value={value}
+              placeholder={placeholder}
+            />
+          </div>
+        );
+      }
+    };
+    return _renderTextInput();
+  }
+}
+
+const defaultClasses = {
```

Comment:
This is not necessary, we are using css modules ;) 

1.- Import your styles
```
import styles from './textfield.scss';
```

2.- Us the styles in the component:

```
<div className={classNames(styles.defaultRoot, classes.root)}>
```

Let me know if you have any questions.